subsys/net/ip/tcp2.c

```diff
@@ -249,6 +249,11 @@ static int tcp_conn_unref(struct tcp *conn)
 
```

Comment:
Very good commit message in `net: tcp2: Implement the retransmission for the outgoing data`